packages/truffle/test/scenarios/commands/console.js

```diff
@@ -1,41 +1,108 @@
 const CommandRunner = require("../commandRunner");
 const Server = require("../server");
-const Config = require("@truffle/config");
 const MemoryLogger = require("../MemoryLogger");
 const assert = require("assert");
+const path = require("path");
+const Reporter = require("../reporter");
+const sandbox = require("../sandbox");
+const tmp = require("tmp");
 
 describe("truffle console", () => {
-  let config;
+  let config, project;
   const logger = new MemoryLogger();
 
   before(async () => await Server.start());
   after(async () => await Server.stop());
 
-  before("before all setup", () => {
-    config = Config.default();
+  project = path.join(__dirname, "../../sources/console");
+
+  before(async function () {
+    this.timeout(10000);
+    config = await sandbox.create(project);
+    config.network = "development";
     config.logger = logger;
+    config.mocha = {
+      reporter: new Reporter(logger)
```

Comment:
You might not need to set this property for all these tests. In other tests I've found this doesn't do anything and I was able to get rid of 
```
config.mocha = {
  reporter: new Reporter(logger)
}
```

Reply:
Yes, sure. I also checked removing those. And it seems that it doesn't do anything. I will get rid of these.